templates/downloads/index.html

```diff
@@ -37,10 +37,61 @@ <h1 class="call-to-action">Download the latest version of Python</h1>
 {% endblock header_content %}
 
 {% block content %}
-                <div class="row download-list-widget">
-
+                <div class="row active-release-list-widget">
                     {% render_active_banner %}
 
+                    <h2 class="widget-title">Active Python Releases</h2>
+                    <p class="success-quote"><a href="https://devguide.python.org/#status-of-python-branches">For more information please visit the Python Developer's Guide</a>.</p>
```

Comment:
Remove "please" for brevity, cf. https://developers.google.com/style/tone#politeness-and-use-of-please

```suggestion
                    <p class="success-quote"><a href="https://devguide.python.org/#status-of-python-branches">For more information visit the Python Developer's Guide</a>.</p>
```